generator/src/main/java/com/scottlogic/deg/generator/fieldspecs/FieldSpec.java

```diff
@@ -34,7 +34,7 @@
     private final MustContainRestriction mustContainRestriction;
     private final FieldSpecSource source;
 
-    private FieldSpec(
+    protected FieldSpec(
```

Comment:
Was the access level changed just to make testing easier?

Reply:
Yes it was, a necessary evil i think.
The mocking framework we have doesn't permit control/verification of `equals`/`hashCode`

Reply:
`FieldSpec` constructor returned to being private.

Reply:
I would imagine that if something is marked as private that means the testing for it is implicit - you would have to work around the access and test whatever calls the private component. Just a unit testing principle I aim to stick to